go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/grafana/terraform-provider-grafana
 go 1.16
 
 require (
-	github.com/grafana/grafana-api-golang-client v0.2.3
+	github.com/grafana/grafana-api-golang-client v0.2.4-0.20220124110757-5a931978a060
```

Comment:
I pushed tag `v0.2.4`. Can you update this and `go mod tidy`?

Reply:
done